### ISSUE TITLE
Can we use colour when printing?

### ISSUE THREAD

**Issue by hadley** [r-lib/xml2#206]
e.g. like https://twitter.com/mdpacer/status/955357848053039104

**Comment by jimhester** [comment]
Yes, very easily with [highlite](https://github.com/jimhester/highlite)

![screenshot 2018-01-23 09 40 57](https://user-images.githubusercontent.com/205275/35281706-962ce3d0-0021-11e8-92e5-978d557780ff.png)

Of course highlite is not on CRAN, but it shouldn't be too hard to get on there, assuming it compiles on Solaris ok...
